cmd/ethereum_rust/ethereum_rust.rs

```diff
@@ -120,16 +120,11 @@ async fn main() {
                 "Adding block {} with hash {:#x}.",
                 block.header.number, hash
             );
-            match add_block(&block, &store) {
```

Comment:
Sorry, maybe let's go back to `add_block`. I know that I suggested the change, but it is all over the place now including documentation, tickets, so maybe we can do this later.

Reply:
Done.